cloud/auth.go

```diff
@@ -14,11 +14,19 @@ import (
 	secretsapi "github.com/earthly/cloud-api/secrets"
 	"github.com/earthly/earthly/util/cliutil"
 	"github.com/earthly/earthly/util/fileutil"
+
+	"github.com/hako/durafmt"
 	"github.com/pkg/errors"
 	"golang.org/x/crypto/ssh"
 	"golang.org/x/crypto/ssh/agent"
 )
 
+const (
+	authExpirationWarningGracePeriod = time.Hour * 24 * 14
```

Comment:
@vladaionescu I set the warning to 14 days or under. Are you good with that or should I set it to 30 days or something else?

Reply:
14 days is reasonable!

Reply:
thanks for the quick reply :)